x/interchainstaking/keeper/zones.go

```diff
@@ -353,13 +353,13 @@ OUTER:
 	}
 
 	// send delegations to validators
-	k.Logger(ctx).Info("send performance delegations", "zone", zone.ChainId)
+	k.Logger(ctx).Info("send performance delegations", "zone", zone.ID())
 
 	msgs := make([]sdk.Msg, len(validatorsToDelegate))
 	for i, val := range validatorsToDelegate {
 		k.Logger(ctx).Info(
 			"performance delegation",
-			"zone", zone.ChainId,
+			"zone", zone.ID(),
```

Comment:
It's a log so not the endo of the world, but performance delegations will only happen on basechain `ChainID()`